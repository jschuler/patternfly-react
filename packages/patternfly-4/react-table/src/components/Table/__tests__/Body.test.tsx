// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { TableBody } from '../Body';

describe('TableBody test', () => {
  it('TableBody should match snapshot', () => {
    const view = shallow(
      <TableBody
        className={"''"}
				children={<>ReactNode</>}
				headerData={undefined}
				rows={undefined}
				rowKey={'id'}
				onRowClick={() => {}}
				onRow={(...args: any) => Object}
      />);
    expect(view).toMatchSnapshot();
  });
});
