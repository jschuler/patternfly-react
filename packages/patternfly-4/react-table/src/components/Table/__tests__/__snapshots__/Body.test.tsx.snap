// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableBody test TableBody should match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
